feature-group-definitions/cascade-layers.yml

```diff
@@ -1,3 +1,11 @@
 spec: https://drafts.csswg.org/css-cascade-5/#layering
 caniuse: css-cascade-layers
 usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/4007
+support_status:
+  baseline:
+    is_baseline: true
```

Comment:
I wonder if we can do something closer to BCD?

```yaml
status:
  baseline: true
```

Adding `experimental` and `deprecated` would then be natural.

Alternatively, a simplification I'd kind of like to see in BCD as well:

```yaml
status: baseline
```

Reply:
I'd like to see something like that too! That said, I immediately began to ask myself:

- What the other values might make up `status` (especially preceding `baseline`)?
- Do any values meaningfully overlap? (suppose `baseline` and `deprecated`, for instance, on the presumption that `baseline` is purely support-driven)?
- What additional data might be needed for those other values? (e.g., do we care, for example, when a feature was formally deprecated?)

I think there's enough questions there that we should not go directly to single-term `status` field. I'm excited at the prospect of getting there eventually, though. Maybe we could reserve a key like `lifecycle` for something like this?

```yaml
lifecycle: proposed | incubating | baseline | deprecated | retired
```

Then it could safely live alongside baseline etc.